bot/utils/setup.py

```diff
@@ -29,13 +30,14 @@ def villager_bot_intents() -> discord.Intents:
     )
 
 
-def load_translations(disabled_translations: list[str]) -> dict[str, Translation]:
+def load_translations() -> dict[str, Translation]:
     translations = dict[str, Translation]()
+    data_file = load_data()
```

Comment:
In the case when `load_translations()` is called in `main_async()`, `load_data()` is called twice. Why wouldn't we just pass in the specific data we need into the the `load_translations()`